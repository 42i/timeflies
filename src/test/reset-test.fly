day 2012-08-23 8.5 17.25, off 0.5
day 2012-08-24 8.75 17.75, off 0.75
day 2012-08-27 8.5 17.25, off 0.5
day 2012-08-28 8.75 17.75, off 0.75
day 2012-08-29 8 18, off 0.5, off 0.25
leave-days 2012-08-30 2012-09-04 # hiking
day 2012-09-05 8.75 17.75, off 0.75
reset
day 2012-09-06 8 18, off 0.5, off 0.25
day 2012-09-07 8.5 17.25, off 0.5
day 2012-09-10 8.75 17.75, off 0.75
day 2012-09-11, sick 8
day 2012-09-12, sick 8
day 2012-09-13 8.5 17.25, off 0.5
day 2012-09-14 8 18, off 0.5, off 0.25
day 2012-09-17 8 18, off 0.5, off 0.25
day 2012-09-18 8.5 17.25, off 0.5
