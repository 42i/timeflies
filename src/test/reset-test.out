Time at work overview (all):
     when        worked   leave    sick    must
2012-08-23 Thu:    8.25 ----.-- ----.--    8.00
2012-08-24 Fri:    8.25 ----.-- ----.--    8.00
  week 2012-34:   16.50 ----.-- ----.--   16.00
2012-08-27 Mon:    8.25 ----.-- ----.--    8.00
2012-08-28 Tue:    8.25 ----.-- ----.--    8.00
2012-08-29 Wed:    9.25 ----.-- ----.--    8.00
2012-08-30 Thu: ----.--    8.00 ----.--    8.00
2012-08-31 Fri: ----.--    8.00 ----.--    8.00
  week 2012-35:   25.75   16.00 ----.--   40.00
 month 2012-08:   42.25   16.00 ----.--   56.00
2012-09-03 Mon: ----.--    8.00 ----.--    8.00
2012-09-04 Tue: ----.--    8.00 ----.--    8.00
    week reset: ----.--   16.00 ----.--   16.00
   month reset: ----.--   16.00 ----.--   16.00
   total reset:   42.25   32.00 ----.--   72.00
2012-09-05 Wed:    8.25 ----.-- ----.--    8.00
2012-09-06 Thu:    9.25 ----.-- ----.--    8.00
2012-09-07 Fri:    8.25 ----.-- ----.--    8.00
  week 2012-36:   25.75 ----.-- ----.--   24.00
2012-09-10 Mon:    8.25 ----.-- ----.--    8.00
2012-09-11 Tue: ----.-- ----.--    8.00    8.00
2012-09-12 Wed: ----.-- ----.--    8.00    8.00
2012-09-13 Thu:    8.25 ----.-- ----.--    8.00
2012-09-14 Fri:    9.25 ----.-- ----.--    8.00
  week 2012-37:   25.75 ----.--   16.00   40.00
2012-09-17 Mon:    9.25 ----.-- ----.--    8.00
2012-09-18 Tue:    8.25 ----.-- ----.--    8.00
  week 2012-38:   17.50 ----.-- ----.--   16.00
 month 2012-09:   69.00 ----.--   16.00   80.00
         total:   69.00 ----.--   16.00   80.00
     when        worked   leave    sick    must
